pkg/app/piped/planpreview/builder.go

```diff
@@ -230,6 +236,11 @@ func (b *builder) build(ctx context.Context, id string, cmd model.Command_BuildP
 	return results, nil
 }
 
+type diffResult struct {
+	Summary  string
+	NoChange bool
```

Comment:
Is there any plan to use it across other packages? If not I feel it's enough to be private fields.

Reply:
Sure. It will only be used inside this package. Just my habit for the private struct.
I am ok to change them. 👍